src/rust/engine/process_execution/src/remote.rs

```diff
@@ -20,6 +20,8 @@ use sha2::Sha256;
 use super::{ExecuteProcessRequest, FallibleExecuteProcessResult};
 use std::cmp::min;
 
+const CACHE_KEY_GEN_VERSION_ENV_VAR_NAME: &str = "PANTS_CACHE_KEY_GEN_VERSION";
```

Comment:
It's apparent when we use it below, but I'd appreciate a comment here along the lines of: "We store this constant to make sure no env variables are define with this name"

Reply:
Done